typing/src/main/java/org/navgurukul/typingguru/utils/TypingGuruPreferenceManager.kt

```diff
@@ -0,0 +1,31 @@
+package org.navgurukul.typingguru.utils
+
+import android.content.Context
+import android.content.SharedPreferences
+
+object TypingGuruPreferenceManager {
+    private const val PREF_NAME = "typing-guru"
+    private var mSharedPreferences: SharedPreferences? = null
+
+    fun init(context: Context) {
+        if (mSharedPreferences == null) {
```

Comment:
This should be provided by koin. As a general rule, we try to delegate instantiating logic outside of classes to make our code more testable